File: Service/PrismaWEB.Domain/Entities/Atividade.cs
namespace ProjetoModeloDDD.Domain.Entities
{
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    public class Atividade
    {
        public int Id { get; set; }

        [Required]
        public DateTime DataRealizada { get; set; }

        [Required]
        public int Pessoa_Id { get; set; }
        [ForeignKey("Pessoa_Id")]
        public virtual Pessoa Pessoa { get; set; }

        [Required]
        public int Tarefa_Id { get; set; }
        [ForeignKey("Tarefa_Id")]
        public virtual Tarefa Tarefa { get; set; }

        [Required]
        public DateTime DataCriacao { get; set; }
        public DateTime? DataAlteracao { get; set; }
    }
}

File: Client/src/models/Atividade.js
export default class Atividade {
  constructor(Id, DataRealizada, Pessoa, Tarefa) {
    this.Id = Id
    this.DataRealizada = this._GetDateByDateIsObjectOrString(DataRealizada)
    this.Pessoa = Pessoa
    this.Tarefa = Tarefa
  }

  _GetDateByDateIsObjectOrString(DataRealizada) {
    if (typeof DataRealizada == "object") return new Date(DataRealizada)
    return new Date(parseInt(DataRealizada.slice(6, DataRealizada.length - 2)))
  }
}
